### PR TITLE
Using checkers assert for integration-cli/docker_test_vars.go

### PR DESCRIPTION
**Pull request by wenchma**
Fixes part of #16756

Signed-off-by: Wen Cheng Ma wenchma@cn.ibm.com

### CODE REVIEW

File: integration-cli/docker_test_vars.go [hunk 1]
```diff
@@ -62,10 +63,7 @@ func init() {
 	}
 	var err error
 	dockerBinary, err = exec.LookPath(dockerBinary)
-	if err != nil {
-		fmt.Printf("ERROR: couldn't resolve full path to the Docker binary (%v)", err)
-		os.Exit(1)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("ERROR: couldn't resolve full path to the Docker binary (%v)", err))
```
Comment (HuKeping) [inline integration-cli/docker_test_vars.go]: the err in comment is no need since it will be printed out anyway.

Comment (wenchma) [inline integration-cli/docker_test_vars.go]: @HuKeping will remove the err in comment

**Comment by HuKeping** [comment]
Please reformat the above files using "gofmt -s -w" and commit the result.

**Comment by wenchma** [comment]
@HuKeping , done, thx

**Comment by vdemeester** [comment]
@wenchma I'm pretty sure there is nothing to do on `docker_test_vars.go`. The change you made is on an `init()` function so there is no `c *check.C` object, and it's there to fail pretty early in case we don't have a `docker` binary.

**Comment by HuKeping** [comment]
@vdemeester is right @wenchma , would you mind closing this?

**Comment by vdemeester** [comment]
Discussed a bit on IRC with @wenchma, Closing it :wink: 
Thanks a lot for your contribution @wenchma, there will be other opportunities to contibrute :stuck_out_tongue:

**Comment by wenchma** [comment]
@vdemeester , thank you! :wink:

**Pull request closed without merging**